### DISCUSSION TITLE
Define supported Postgres versions for Postgres Plugin

### DISCUSSION THREAD

**Discussion by dylanlott** [ConduitIO/conduit#177] [General]
We currently do not have a minimum supported version defined for the Postgres plugin. We should define a minimum and preferred Postgres version, and be explicit about what operation modes are supported in which versions.

**Comment by devarismeroxa** [comment]
What is the minimum version that can have CDC enabled either native or via a plugin? That would be my starting point @dylanlott

**Reply by neovintage** [reply]
I'd start with what most cloud providers are choosing to support. For example, AWS RDS is about to deprecate 9.6. This means 10 will be the oldest supported version.

**Reply by dylanlott** [reply]
@devarismeroxa I believe version 9.4 introduced access to the WAL, which enabled CDC features.
Supporting what most other cloud providers are supporting makes sense to me, though, especially if we don't have a specific need for an older version.

**Reply by dylanlott** [reply]
[Postgres versioning](https://www.postgresql.org/support/versioning/) has `10.19` as the latest minor version of `10` and it has a scheduled final release of 10 Nov 2022. If we pick `10`, my thought would be latest minor version of `10`, but I'm open to other ideas.

For additional context, the final release for a `9.X` version was 21 Nov 2021.

**Reply by neovintage** [reply]
@dylanlott v10 sounds reasonable to me. If there are a ton of requests for lower versions we could revisit. :shipit:

**Reply by dylanlott** [reply]
@neovintage I'm going to update the documentation to reflect this version choice then. 👍
